### PR TITLE
Update server-side properties with recent changes

### PR DESCRIPTION
**Pull request by ajbozarth**
While #1620 was in developement there was a couple changes made to
the propeties files on the front end that were missed when moving
the content to the backend (#1631 and #1654).

This adds back those missing changes.

This brings back the refresh env vars button that was lost.

### How was this pull request tested?
Tested locally

 
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

**Comment by ajbozarth** [comment]
Found this while testing #1779

File: elyra/pipeline/resources/properties.json [hunk 1]
```diff
@@ -67,7 +67,7 @@
           "default": "Filename"
         },
         "description": {
-          "default": "The path to the notebook file.",
+          "default": "The path to the file.",
```
Comment (bourdakos1) [inline elyra/pipeline/resources/properties.json:70]: I think this is dynamically handled in the server code

Comment (ajbozarth) [reply elyra/pipeline/resources/properties.json:70]: I saw that, so I figured this a default/fallback value, so it should be generic. I changed this before finding that dynamic handling and just didn't change it back for that reason.

**Review by bourdakos1 — APPROVED** [review]

**Review by kiersten-stokes — COMMENTED** [review]
Quick question -- when I remove an env var from a file, save it, and then go back to the properties to hit refresh, the removed value persists. Is this the functionality that we ultimately decided on? I vaguely remember a conversation about this. Happens with all file types.

Not sure if this is worth nitpicking right now though. Refresh works exactly as expected in the rest of the scenarios that I tested.

**Comment by ajbozarth** [comment]
> Is this the functionality that we ultimately decided on?

This is the intended functionality, there was discussion about pruning unset vars that are no longer in the file, but that kind of validation was decided to be dealt with later alongside the new properties service we were planning

**Review by kiersten-stokes — APPROVED** [review]
LGTM!

**Review by lresende — APPROVED** [review]

**Review by lresende — CHANGES_REQUESTED** [review]
@ajbozarth Please check test failures

**Comment by ajbozarth** [comment]
I've pushed a fix for the test failure, though it makes me think the test is written poorly rather than properly failing. It checks that the api returns the same content as the json file on the backend. But the backend code always edits that content before returning. This test only currently passes because the content specific to the notebook endpoint is the default values in the file. I broke it by changing a default value to be generic. @lresende you wrote the test, was this intentional or should the test be updated? If so we should do that in a separate issue anyway.

**Comment by lresende** [comment]
I did not look at what was failing? What test was it? Feel free to enhance tests at ay time :)

**Review by lresende — APPROVED** [review]

**Pull request merged**